addons/account/i18n/account.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
```

Comment:
don't know what is the convention for that kind of stuff, do i do all the po files because the pot by itself is a bit useless but do i really want to do 45 po files ahaha

Reply:
According to the guide, we should only add the `.po` files only if it's on stable and done in a `l10n_*` module. Since this is not l10n, I think we're good like this :)